composer.json

```diff
@@ -85,6 +85,7 @@
 		"test-integration-adminonly": "\"vendor/bin/phpunit\" --testsuite integration --colors=always --configuration tests/Integration/phpunit.xml.dist --group AdminOnly",
 		"test-integration-docloudflare": "\"vendor/bin/phpunit\" --testsuite integration --colors=always --configuration tests/Integration/phpunit.xml.dist --group DoCloudflare",
 		"test-integration-withwoo": "\"vendor/bin/phpunit\" --testsuite integration --colors=always --configuration tests/Integration/phpunit.xml.dist --group WithWoo",
+		"test-integration-cpcss": "\"vendor/bin/phpunit\" --testsuite integration --colors=always --configuration tests/Integration/phpunit.xml.dist --group CriticalCss",
```

Comment:
This script isn't needed. Why? We only want to add a script for a separate test _when_ we need to do a different bootstrap set up or run in a separate process.

Locally, we can isolate and run the groups we want by doing this:

```
composer test-integration -- --group CriticalCss
```

That `--` lets composer know we are adding new arguments.